src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -447,6 +471,17 @@ protected function initializeBundles()
     }
 
     /**
+     * The extension point similar to the Bundle::build() method.
+     *
+     * Use this method to register compiler passes and manipulate the container during the building process.
+     *
+     * @param ContainerBuilder $container
+     */
+    protected function build(ContainerBuilder $container)
```

Comment:
I don't like the name of this method. According to the [Symfony naming conventions](http://symfony.com/doc/current/contributing/code/conventions.html) I think this should be called `buildContainer()` because we war not building the kernel, which is what the `Kernel` class is about.